tests/utils/dir-is-empty.test.js

```diff
@@ -0,0 +1,9 @@
+test('dir is empty', () => {
+    const dirIsEmpty = require('utils/dir-is-empty.js');
+    expect(!dirIsEmpty(process.cwd())).toBeTruthy();
+});
+
+test('dir is not empty', () => {
```

Comment:
each test case should exercise the code differently - this test case tests exactly what the previous test case did - namely that the dirIsEmpty works correctly for a directory that is known to have contents.

This test case should create a temp directory - which is known to be empty - then test that dirIsEmpty returns false on that directory.    The test case should clean up that directory on successful completion.